types/trace-events.d.ts

```diff
@@ -0,0 +1,590 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+// https://github.com/Hoten/chrome-trace-events-tsc/blob/master/dist/lh-trace-events.d.ts
```

Comment:
I'm not sure of the best way to do this. Publish separately and import (like we do with the protocol types), pull into core, or some kind of third_party thing?

It could be fine to have the file here and just have a comment saying to regenerate go run this script, but the original probably should switch to an explicit Apache 2.0 license (currently it only has MIT in `package.json`).

Reply:
I think we should just pull from the other repo as-needed. I'd like to defer publishing to npm until a project other than LH uses it.